pallets/dkg-proposal-handler/src/lib.rs

```diff
@@ -13,12 +13,11 @@ mod mock;
 
 #[cfg(test)]
 mod tests;
-
 use dkg_runtime_primitives::{
-	Address, ChainIdTrait, ChainIdType, DKGPayloadKey, EIP1559TransactionMessage,
-	EIP2930TransactionMessage, LegacyTransactionMessage, OffchainSignedProposals, Proposal,
-	ProposalAction, ProposalHandlerTrait, ProposalHeader, ProposalKind, ProposalNonce,
-	TransactionV2, OFFCHAIN_SIGNED_PROPOSALS,
+	offchain::storage_keys::OFFCHAIN_SIGNED_PROPOSALS, Address, ChainIdTrait, ChainIdType,
+	DKGPayloadKey, EIP1559TransactionMessage, EIP2930TransactionMessage, LegacyTransactionMessage,
+	OffchainSignedProposals, Proposal, ProposalAction, ProposalHandlerTrait, ProposalHeader,
+	ProposalKind, ProposalNonce, TransactionV2,
```

Comment:
I don't see `Address`, `ProposalNonce`, `ProposalHeader`, and `TransactionV2` being used, I think they can be removed.